src/themes/default/tooltip/hover.css

```diff
@@ -0,0 +1,78 @@
+.tooltip-content {
```

Comment:
Note: I move those into the global stylesheet for re-use. Plus fixed:
- references to non-existing color variables used in the Radix demo
- missing animation keyframe settings